k8s.gcr.io/images/k8s-staging-capi-openstack/images.yaml

```
@@ -11,3 +11,4 @@
     "sha256:f9461a1cafb2562e2efea8e4829b75a93d59fd60923d369528766cb10f490076": ["v0.4.0"]
     "sha256:12539f635df1db226fc72619e7cf0b901cc4f1cd4ae26b08d05d7d14ee482209": ["v0.5.0"]
     "sha256:098c626a57fbaca6d75a0b9db7a410e88091ef74c37076554192f942d6368bf9": ["v0.5.1"]
+    "sha256:037a9d5e820d1b462b2e531028d560b87028b5b76fa7c1d5ba5b5ba7b2c15ffb": ["v0.5.2"]
```

```suggestion
    "sha256:5c76dea2dd7d6b62034eb7df1a38d72290a1e33fd9528db372f5a745807dc74d": ["v0.5.2"]
```

I think it should be this sha (cf. https://console.cloud.google.com/gcr/images/k8s-staging-capi-openstack/global/capi-openstack-controller@sha256:5c76dea2dd7d6b62034eb7df1a38d72290a1e33fd9528db372f5a745807dc74d/details)